pkg/sql/information_schema.go

```diff
@@ -262,6 +263,28 @@ https://www.postgresql.org/docs/9.5/infoschema-applicable-roles.html`,
 	},
 }
 
+var informationSchemaCharacterSets = virtualSchemaTable{
+	comment: `character sets available in the current database
+` + docs.URL("information-schema.html#character_sets") + `
+https://www.postgresql.org/docs/9.5/infoschema-character-sets.html`,
+	schema: vtable.InformationSchemaCharacterSets,
+	populate: func(ctx context.Context, p *planner, dbContext *dbdesc.Immutable, addRow func(...tree.Datum) error) error {
+		return forEachDatabaseDesc(ctx, p, nil /* all databases */, true, /* requiresPrivileges */
+			func(db *dbdesc.Immutable) error {
+				return addRow(
+					tree.DNull,                    // character_set_catalog
+					tree.DNull,                    // character_set_schema
+					tree.NewDString("UTF8"),       // character_set_name: UTF8 is the only available encoding
+					tree.NewDString("UCS"),        // character_repertoire: UCS for UTF8 encoding
+					tree.NewDString("UTF8"),       // form_of_use: same as the database encoding
+					tree.NewDString(db.GetName()), // default_collate_catalog
+					tree.NewDString("pg_catalog"), // default_collate_schema
```

Comment:
regarding the last two columns - postgresql leaves those empty; are you sure that populating them will not crate an incompatibility?

Reply:
do they?

```

 otan@otan-mbp  ~/go/src/github.com/cockroachdb/cockroach   srid_from_geojson  psql
psql (13.0)
Type "help" for help.

otan=# select * from information_schema.character_sets;
 character_set_catalog | character_set_schema | character_set_name | character_repertoire | form_of_use | default_collate_catalog | default_collate_schema | default_collate_name
-----------------------+----------------------+--------------------+----------------------+-------------+-------------------------+------------------------+----------------------
                       |                      | UTF8               | UCS                  | UTF8        | otan                    | pg_catalog             | ucs_basic
```

(happy to change this in a subsuquent PR if so)

Reply:
I have this:
```
kena@kenax ~/cockroach % psql
psql (11.10)
Type "help" for help.

kena=> table information_schema.character_sets;
 character_set_catalog | character_set_schema | character_set_name | character_repertoire | form_of_use | default_collate_catalog | default_collate_schema | default_collate_name
-----------------------+----------------------+--------------------+----------------------+-------------+-------------------------+------------------------+----------------------
                       |                      | UTF8               | UCS                  | UTF8        | kena                    |                        |
(1 row)
```